operator/pkg/central/extensions/reconcile_central_db_password.go

```diff
@@ -67,7 +67,7 @@ func (r *reconcileCentralDBPasswordExtensionRun) readAndSetPasswordFromReference
 
 func (r *reconcileCentralDBPasswordExtensionRun) Execute(ctx context.Context) error {
 	if r.centralObj.DeletionTimestamp != nil {
-		return r.DeleteSecret(ctx, canonicalCentralDBPasswordSecretName)
+		return nil // do not delete the secret as the PVC is not deleted
```

Comment:
Question? Should we delete the secret once the PVC is deleted ?

Reply:
That might be a nice touch, but I cannot think of a reliable way to do this...